### PR TITLE
Ghost Marker animated

### PR DESCRIPTION
**Pull request by AdamTadeusz**
in-code ghost marker animation

### CODE REVIEW

File: mp/src/game/client/neo/ui/neo_hud_ghost_marker.cpp [hunk 1]
```diff
@@ -111,31 +111,49 @@ void CNEOHud_GhostMarker::DrawNeoHudElement()
...
+			{
+				// If viewing from playing player, but opposite of ghosting team, show red
+				ghostColor = COLOR_RED;
+			}
+			else
+			{
+				// Otherwise show ghosting team color (if friendly or spec)
+				ghostColor = (m_iGhostingTeam == TEAM_JINRAI) ? COLOR_JINRAI : COLOR_NSF;
+			}
 		}
```
Comment (nullsystem) [inline mp/src/game/client/neo/ui/neo_hud_ghost_marker.cpp:146]: These will override the alpha values. Set the initial colors then edit the alpha value on the color.

**Review by nullsystem — CHANGES_REQUESTED** [review]

File: mp/src/game/client/neo/ui/neo_hud_ghost_marker.cpp (hunk 1 above)
Comment (AdamTadeusz) [reply mp/src/game/client/neo/ui/neo_hud_ghost_marker.cpp]: yeah thats why i marked it as a draft but thanks xd

Comment (nullsystem) [reply mp/src/game/client/neo/ui/neo_hud_ghost_marker.cpp]: I was thinking more like `ghostColor[3] = alpha` so the colors can still be used but it's the same result.

**Review by nullsystem — APPROVED** [review]

File: mp/src/CMakeLists.txt [hunk 2]
```diff
@@ -131,6 +131,7 @@ if(OS_WINDOWS)
         /errorReport:prompt # ErrorReporting - Prompt Immediately
         /W4
         /Gw # Optimize Global Data
+        /ZI
```
Comment (nullsystem) [inline mp/src/CMakeLists.txt:134]: Actually, don't commit this change in. MSVC is just blasting with warnings.

**Review by nullsystem — CHANGES_REQUESTED** [review]

File: mp/src/game/client/neo/ui/neo_hud_ghost_marker.cpp [hunk 3]
```diff
@@ -129,13 +124,38 @@ void CNEOHud_GhostMarker::DrawNeoHudElement()
...
 	}
 
-	surface()->DrawSetColor(ghostColor);
-	surface()->DrawSetTexture(m_hTex);
-	surface()->DrawTexturedRect(
-		offset_X,
-		offset_Y,
-		offset_X + (m_iMarkerTexWidth * scale),
-		offset_Y + (m_iMarkerTexHeight * scale));
+	const float scale = neo_ghost_marker_hud_scale_factor.GetFloat();
```
Comment (nullsystem) [inline mp/src/game/client/neo/ui/neo_hud_ghost_marker.cpp:127]: Also, change the default of the scale factor. It looked very small with 0.125 default.

**Review by nullsystem — CHANGES_REQUESTED** [review]

**Review by nullsystem — APPROVED** [review]

**Pull request merged**